pkg/basicstation/cups/update_info.go

```diff
@@ -314,16 +308,19 @@ func (s *Server) UpdateInfo(c echo.Context) (err error) {
 			res.LNSURI = fmt.Sprintf("%s://%s", scheme, address)
 		}
 
-		lnsTrust, err := s.getTrust(gtw.GatewayServerAddress)
-		if err != nil {
-			return errServerTrust.WithCause(err).WithAttributes("address", gtw.GatewayServerAddress)
-		}
-		lnsCredentials, err := TokenCredentials(lnsTrust, string(gtw.LbsLnsSecret.Value))
-		if err != nil {
-			return err
-		}
-		if crc32.ChecksumIEEE(lnsCredentials) != req.LNSCredentialsCRC {
-			res.LNSCredentials = lnsCredentials
+		// Only fetch Trust and Credentials for TLS end points.
+		if scheme != "ws" {
```

Comment:
Shouldn't we just check for `wss` here? Otherwise we'd also run this if anyone decided to put `tcp://` for the scheme.